src/js/edu-benefits/utils/validations.js

```diff
@@ -287,108 +197,32 @@ function isValidSpouseInformation(data) {
-    if (!isValidIncome(children[i])) {
-      return false;
-    }
-  }
   return true;
 }
 
-function isValidAnnualIncome(data) {
-  let isValidSpouseIncomeFields = true;
-
-  if (data.spouseGrossIncome && data.spouseNetIncome && data.spouseOtherIncome) {
-    isValidSpouseIncomeFields =
-      isValidField(isValidMonetaryValue, data.spouseGrossIncome) &&
-      isValidField(isValidMonetaryValue, data.spouseNetIncome) &&
-      isValidField(isValidMonetaryValue, data.spouseOtherIncome);
-  }
-
-  return isValidField(isValidMonetaryValue, data.veteranGrossIncome) &&
-    isValidField(isValidMonetaryValue, data.veteranNetIncome) &&
-    isValidField(isValidMonetaryValue, data.veteranOtherIncome) &&
-    isValidSpouseIncomeFields &&
-    isValidChildrenIncome(data.children);
-}
-
-function isValidDeductibleExpenses(data) {
-  return isValidField(isValidMonetaryValue, data.deductibleMedicalExpenses) &&
-    isValidField(isValidMonetaryValue, data.deductibleFuneralExpenses) &&
-    isValidField(isValidMonetaryValue, data.deductibleEducationExpenses);
-}
-
-function isValidVAFacility(data) {
-  return validateIfDirty(data.facilityState, isNotBlank) &&
-    validateIfDirty(data.vaMedicalFacility, isNotBlank);
-}
-
-function isValidMedicareMedicaid(data) {
-  let isValidEffectiveDate = true;
-
-  if (data.isEnrolledMedicarePartA.value === 'Y') {
-    isValidEffectiveDate = isValidDateField(data.medicarePartAEffectiveDate);
-  }
-
-  return validateIfDirty(data.isMedicaidEligible, isNotBlank) &&
-    validateIfDirty(data.isEnrolledMedicarePartA, isNotBlank) &&
-    isValidEffectiveDate;
+function isTourOfDutyValid(tour) {
+  return isNotBlank(tour.serviceBranch.value)
+    && isValidDateField(tour.fromDate)
+    && isValidDateField(tour.toDate)
+    && isValidSeparatedDateField(tour.fromDate, tour.toDate);
 }
 
-function isValidGeneralInsurance(data) {
-  let allProvidersValid = true;
-  const providers = data.providers;
-
-  for (let i = 0; i < providers.length; i++) {
-    if (!(isNotBlank(providers[i].insuranceName.value) &&
-        isNotBlank(providers[i].insurancePolicyHolderName.value) &&
-        isValidInsurancePolicy(providers[i].insurancePolicyNumber.value, providers[i].insuranceGroupCode.value))
-    ) {
-      allProvidersValid = false;
-    }
-  }
-
-  return isNotBlank(data.isCoveredByHealthInsurance.value) &&
-      allProvidersValid;
-}
-
-function isValidServiceInformation(data) {
-  return isNotBlank(data.lastServiceBranch.value) &&
-      (isValidDateField(data.lastEntryDate) && isValidEntryDateField(data.lastEntryDate, data.veteranDateOfBirth)) &&
-      (isValidDateField(data.lastDischargeDate) && isValidDischargeDateField(data.lastDischargeDate, data.lastEntryDate)) &&
-      isNotBlank(data.dischargeType.value);
-}
-
-function isBenefitsInformationSectionValid(data) {
-  return !data.chapter33 || isNotBlank(data.benefitsRelinquished.value);
+function isMilitaryServicePageValid(data) {
```

Comment:
I think we need another validation function for the MilitaryServiceTour questions. Right now if you try to add a period of service and then save it, it doesn't register it as valid.